gtest_parallel.py

```diff
@@ -577,8 +577,8 @@ def find_tests(binaries, additional_args, options, times):
       command += ['--gtest_also_run_disabled_tests']
 
     list_command = command + ['--gtest_list_tests']
-    if options.gtest_filter != '':
-      list_command += ['--gtest_filter=' + options.gtest_filter]
+    if options.test != '':
```

Comment:
Can we have both?

Reply:
You can: if I pass `--size=S --test=Pipeline*`, I get all `Pipeline*` tests of size Small

Reply:
Or are you talking about both `--gtest_filter` and `--test`? We can have both of those, too.

Reply:
Sorry. Yes to both `--gtest_filter` and `--test`